app/helpers/ontology_helper.rb

```diff
@@ -13,4 +13,53 @@ def download_path(resource)
   def show_evaluate?
     show_oops? #|| show_foo?
   end
+
+  def status_tag(resource)
```

Comment:
This is quite old code that has been changed (also the method has been renamed to `state_tag`) in several other pull requests. Seems like it got back in with a rebase.